src/SOS/Strike/strike.cpp

```diff
@@ -16189,31 +16137,19 @@ HRESULT SetNGENCompilerFlags(DWORD flags)
 {
     HRESULT hr;
 
-    ToRelease<ICorDebugProcess2> proc2;
-    ICorDebugProcess* pCorDebugProcess;
-    if (FAILED(hr = g_pRuntime->GetCorDebugInterface(&pCorDebugProcess)))
-    {
-        ExtOut("SOS: warning, prejitted code optimizations could not be changed. Failed to load ICorDebug HR = 0x%x\n", hr);
-    }
-    else if (FAILED(pCorDebugProcess->QueryInterface(__uuidof(ICorDebugProcess2), (void**) &proc2)))
+    // if CLR is already loaded, try to change the flags now
+    hr = GetTarget()->GetRuntime(&g_pRuntime);
```

Comment:
hmm. Shouldn't initialization be done in a common path instead of initializing the runtime every time?

Reply:
This is a special case because SetNGENCompilerFlags gets called from the EventCallbacks::LoadModule and it is historical. Normally for commands that use the INIT_API_EXT macro will get g_pRuntime setup.  

This eventually could be cleaned up but I don't want to do it in this PR.